Package.swift

```diff
@@ -16,7 +16,8 @@ let package = Package(
         .package(url: "https://github.com/soto-project/soto.git", from: "6.8.0"),
         .package(url: "https://github.com/sebsto/CLIlib/", from: "0.1.2"),
         .package(url: "https://github.com/adam-fowler/swift-srp", from: "2.1.0"),
-        .package(url: "https://github.com/apple/swift-crypto", from: "3.12.3"),        
+        .package(url: "https://github.com/apple/swift-crypto", from: "3.12.2"),
+        .package(url: "https://github.com/apple/swift-testing", from: "0.3.0"),
```

Comment:
Swift testing is included in Swift 6. Migrate the Package.swift to use Swift 6 language mode and remove the dependency on Swift testing